root/main/index.js

```diff
@@ -192,10 +204,19 @@ const ReleaseArtwork = ({artwork}: {|+artwork: ArtworkT|}) => {
             entity: release.name,
           })}
         >
-          <ArtworkImage
-            artwork={artwork}
-            fallback={release.cover_art_url || ''}
-          />
+          {withSafeBrowsing ? (
+            <span className="cover-art-image">
+              {l('Image temporarily disabled until {link|issue CAA-116} gets solved', {
```

Comment:
I'd suggest replacing the entire artwork div (instead than each specific artwork box) with a text such as "Recent artwork has been disabled for Chrome users due to overzealous Google Safe Browsing phishing alerts. Our cover art is stored in Internet Archive servers, and while those servers might also have been used to host dubious content (leading to a server-wide Safe Browsing alert), the images themselves are perfectly safe. Images have only been disabled in the front page; if you get a "Deceptive site ahead" alert elsewhere while browsing MusicBrainz, you can ignore it by clicking "Details" and then "visit this unsafe site". You can follow {link|issue CAA-116} to be informed about any changes to this situation."

Reply:
I still like the idea to show spots missing images. Could your longer text be put in a dedicated banner (at the top of the page) that shows up with Chrome-based browsers only?

Reply:
What do we win by showing the spots? If you want, we can do both, but I really don't see the point tbh :/

Reply:
I have a third take. :) I'd rather just show some clickable img alt text for Chrome users, like "Release by Artist," as I don't see a reason to remove or cripple the feature entirely. I realize the main purpose is to showcase some nice artwork on the front page, but it's still somewhat interesting as a live feed of sorts showing activity on the site. And we don't know how long we'll need it like this, so the less obnoxious the better.

Then I'd put reosarevok's explanation above it, or copy it to a doc page and link it as "Why aren't images showing here?"

Reply:
As long as the explanation is clearly visible, so that when they find a red screen of bullcrap elsewhere on the site they can maybe understand what's going on, I like that idea.